XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/Simulation/DefaultMixedRealityInputSimulationDataProvidersProfile.asset

```diff
@@ -13,4 +13,11 @@ MonoBehaviour:
   m_Name: DefaultMixedRealityInputSimulationDataProvidersProfile
   m_EditorClassIdentifier: 
   isCustomProfile: 0
-  registeredInputSimulationDataProviders: []
+  registeredInputSimulationDataProviders:
+  - dataProviderType:
+      reference: XRTK.Providers.InputSystem.Simulation.CameraSimulationDataProvider,
+        XRTK
+    dataProviderName: Camera Input Simulation Data Provider
```

Comment:
What exactly is `Camera Input`?

Reply:
The naming is maybe not perfect. In MRTK it was called "ManualCameraControl". It's basically not input but simulated head / camera movement using keyboard and mouse. So we are not simulating some sort of camera input / video stream or whatever but we are simulating movement of the camera here.

Reply:
Does that help?

Reply:
I'd agree the naming probably doesn't help, as in the scenario you describe, the camera is an Actor, not an Input, as in, we are taking input to affect the camera.

Are we likely over complicating this taking directly what MRTK implemented, is this simply not just a special controller which can have a camera attached (remembering, Unity can have multiple cameras in a scene)

Does that make sense?

Reply:
If the camera is receiving input from some input data provider then this should be a handler not a service.